### PR TITLE
Editor update to U2017.2.0f1

### PR DESCRIPTION
**Pull request by StephenHodgson**
Updated to U2017.2.0f1

duplicate #1012

### CODE REVIEW

File: Assets/HoloToolkit/Input/Scripts/InputManager.cs [hunk 1]
```diff
@@ -334,7 +334,6 @@ public void HandleEvent<T>(BaseEventData eventData, ExecuteEvents.EventFunction<
                 GameObject fallbackInput = fallbackInputStack.Peek();
                 if (ExecuteEvents.ExecuteHierarchy(fallbackInput, eventData, eventHandler) && eventData.used)
                 {
-                    return;
```
Comment (keveleigh) [inline Assets/HoloToolkit/Input/Scripts/InputManager.cs]: Why was this removed?

Comment (StephenHodgson) [reply Assets/HoloToolkit/Input/Scripts/InputManager.cs]: Good question.

Comment (keveleigh) [reply Assets/HoloToolkit/Input/Scripts/InputManager.cs]: I'll re-add it in my incoming merge PR

Comment (keveleigh) [reply Assets/HoloToolkit/Input/Scripts/InputManager.cs]: I guess it doesn't matter too much, since this is the end of the method either way. For consistency's sake, I'm re-adding it.

Comment (StephenHodgson) [reply Assets/HoloToolkit/Input/Scripts/InputManager.cs]: Oh I see. It was the last return, which was redundant, and my formatting tool removed it.
